include/edl/sgx_ecdsa.edl

```diff
@@ -25,7 +25,7 @@ enclave {
                                                                 [in, string] const char *name,
                                                                 [in, size=evidence_len] attestation_evidence_t *evidence,
                                                                 uint32_t evidence_len,
-                                                                [in, size=hash_len] uint8_t *hash,
+                                                                [in, size=hash_len] const uint8_t *hash,
```

Comment:
please spilt one commit about `adding const about hash value`

Reply:
Please split this pr into several commits. The smaller the function of each commit, the better.